hummingbot/hummingbot
# feat/add moving_price_band and config on the fly for price_band and price_ceiling

Before submitting this PR, please make sure:



A description of the changes proposed in the pull request:
Add feature based on request https://github.com/hummingbot/hummingbot/issues/5142
price_floor and price_ceiling are still useful in cases where prices are stable or as a hard limit such as for stable coin, hence I did not remove it and instead added them as a new parameters instead
1. Add moving_price_band feature to set price band at a percentage of current price every price_band_refresh_Time
2. Update price_floor and price_ceiling to be config able on the fly

Tests performed by the developer:
1. Verify price_floor and price_ceiling is updated
2. Added unit test to verify price_floor and price ceiling percentage is correct
3. Verify that all new params and price_floor and price_ceiling can config on the fly

Tips for QA testing:
)
